topbeat/docs/fields.asciidoc

```diff
@@ -169,6 +169,78 @@ type: int
 The amount of CPU time spent in involuntary wait by the virtual CPU while the hypervisor was servicing another processor. Available only on Unix.
 
 
+=== cpus Fields
+
+This group contains cpu usage per core statistics.
+
+
+=== cpui Fields
+
+This group contains cpu usage statistics of the core i, where i is the core number
```

Comment:
I think it would look better to use cpuX with X being the variable rather then "i".